domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/service/ReviewScheduleDateCalculationService.kt

```diff
@@ -37,6 +37,7 @@ class ReviewScheduleDateCalculationService {
     private const val EXEMPTION_ADDITIONAL_DAYS = 5L
     private const val EXCLUSION_ADDITIONAL_DAYS = 10L
     private const val SYSTEM_OUTAGE_ADDITIONAL_DAYS = 5L
+    private const val RESCHEDULE_ADDITIONAL_DAYS = 10L
```

Comment:
10 days agreed with Aliki . She is seeing this scenario as very similar to a transfer, so 10 days is the same as what happens for a transfer 👍